CloudFoundry/public/apps/index.xml

```diff
@@ -2,17 +2,17 @@
 <rss version="2.0" xmlns:atom="http://www.w3.org/2005/Atom">
   <channel>
     <title>Apps on Govt PaaS</title>
-    <link>http://documentation.trial.cf.paas.alphagov.co.uk/apps/</link>
+    <link>http://localhost:1313/apps/</link>
```

Comment:
Are these changes intended?  I'm not sure what these XML files are used for, so I don't know whether this change matters or not...